spring-cloud-netflix-eureka-client/src/test/java/org/springframework/cloud/netflix/eureka/http/RestClientTransportClientFactoryTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.eureka.http;
+
+import com.netflix.discovery.shared.resolver.DefaultEndpoint;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.web.client.RestClient;
+
+/**
+ * @author Wonchul Heo
+ */
+class RestClientTransportClientFactoryTest {
+
+	private RestClientTransportClientFactory transportClientFactory;
+
+	@BeforeEach
+	void setup() {
+		transportClientFactory = new RestClientTransportClientFactory(RestClient::builder);
+	}
+
+	@Test
```

Comment:
All the tests methods need some kind of assertions to verify that whatever was expected to happen has, in fact, happened.